syntax="proto3";
package lwz_example;
service model{
    rpc lwz_show(lwz_Request) returns (lwz_Response){}
}
 
 
message lwz_Request {
    string index = 1;
    string name = 2;
    int64 age = 3;
}
 
message lwz_Response{
    string stu_class = 1;
    int64 grade = 2;
}
